### ISSUE TITLE
!= not works for SubsetQuery table

### ISSUE THREAD

**Issue by EXPEnbansal** [unclebob/fitnesse#1115]
For subset query table != compares only the first result in the list and it does not check if the unexpected result is not there in the whole list.

**Comment by alexrun** [comment]
@EXPEnbansal can you share a use case for this behavior?

**Comment by EXPEnbansal** [comment]
Yes a simple use case is for example through subset unordered queries i get ten rows of single column and I want to verify that none of the returned value should match some value say x.

**Comment by GiorgosMarkogiannis** [comment]
Hi, i would be interested in trying to fix the issue. Could you provide more details regarding the part of the code that relates to it.

**Comment by six42** [comment]
Excellent Giorgos, 

Thanks for taking this up. 

Subset queries are processed by the code below. 
If you can create first  a sample  test page which fails due to this missing feature I might can give you even more details where to look. 

<https://github.com/unclebob/fitnesse/blob/master/src/fitnesse/testsystems/slim/tables/SubsetQueryTable.java>
